<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SteelCncrGUI</class>
 <widget class="QWidget" name="SteelCncrGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>564</width>
        <height>637</height>
       </rect>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>489</width>
         <height>640</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="leftVLayout">
         <item>
          <widget class="QGroupBox" name="nameGBox">
           <property name="title">
            <string>Denominazione</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabelRichName" name="nameLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
Nome
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEditValue" name="nameLEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="standardGBox">
           <property name="title">
            <string>Acciai standard</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Tipologia</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="standardTypeCBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Denominazione</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="standardNameCBox"/>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="loadStandardPButton">
                <property name="text">
                 <string>Carica</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="deformationGBox">
           <property name="title">
            <string>Deformazione</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabelRichName" name="EsLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;
&lt;/head&gt;
&lt;body&gt;
E&lt;span style=&quot; vertical-align:sub;&quot;&gt;s&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEditValue" name="EsLEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabelRichName" name="epsYdLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;
&lt;body&gt;
ε&lt;span style=&quot; vertical-align:sub;&quot;&gt;yd&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEditValue" name="epsYdLEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabelRichName" name="epsUkLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
ε&lt;span style=&quot; vertical-align:sub;&quot;&gt;uk&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEditValue" name="epsUkLEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabelRichName" name="ratioEpsUdEpsUkLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
ε&lt;span style=&quot; vertical-align:sub;&quot;&gt;ud&lt;/span&gt;/ε&lt;span style=&quot; vertical-align:sub;&quot;&gt;uk&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEditValue" name="ratioEpsUdEpsUkLEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabelRichName" name="epsUdLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
ε&lt;span style=&quot; vertical-align:sub;&quot;&gt;ud&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEditValue" name="epsUdLEdit"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabelUnitMeasure" name="EsUMLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;
&lt;/head&gt;
&lt;body&gt;
[Pa]
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabelUnitMeasure" name="epsYdUMLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;
&lt;body&gt;
[-]
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabelUnitMeasure" name="epsUkUMLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[-]
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabelUnitMeasure" name="ratioEpsUdEpsUkUMLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[-]
&lt;/body&gt;
&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabelUnitMeasure" name="epsUdUMLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[-]
&lt;/body&gt;
&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QCheckBoxReadOnly" name="epsUdCheckBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QCheckBoxReadOnly" name="epsYdCheckBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="tensionGBox">
           <property name="title">
            <string>Tensione</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QLabelRichName" name="fykLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
f&lt;span style=&quot; vertical-align:sub;&quot;&gt;yk&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEditValue" name="fykLEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabelRichName" name="ftkLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
f&lt;span style=&quot; vertical-align:sub;&quot;&gt;tk&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEditValue" name="ftkLEdit"/>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabelRichName" name="kLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
k = (f&lt;span style=&quot; vertical-align:sub;&quot;&gt;t&lt;/span&gt;/ f&lt;span style=&quot; vertical-align:sub;&quot;&gt;y&lt;/span&gt;)&lt;span style=&quot; vertical-align:sub;&quot;&gt;k&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEditValue" name="kLEdit"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabelUnitMeasure" name="ftkUMLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[Pa]
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabelUnitMeasure" name="fykUMLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[Pa]
&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabelUnitMeasure" name="kUMLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[-]
&lt;/body&gt;
&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="fykLabelPButton">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="ftkLabelPButton">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="diagramGBox">
         <property name="title">
          <string>Diagramma σ-ε SLU</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QRadioButtonEnum" name="modelULSRB1">
            <property name="text">
             <string>Fig. 4.1.2 a</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButtonEnum" name="modelULSRB2">
            <property name="text">
             <string>Fig. 4.1.2 b</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCustomPlot" name="plotSULS" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="designGBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Progetto</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabelRichName" name="gammaSLabel">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
γ&lt;span style=&quot; vertical-align:sub;&quot;&gt;S&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabelUnitMeasure" name="gammaSUMLabel">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[-]
&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEditValue" name="gammaSLEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabelRichName" name="fydLabel">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
f&lt;span style=&quot; vertical-align:sub;&quot;&gt;yd&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabelUnitMeasure" name="fydUMLabel">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[Pa]
&lt;/body&gt;
&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEditValue" name="fydLEdit"/>
             </item>
             <item row="1" column="3">
              <widget class="QCheckBoxReadOnly" name="fydCheckBox">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabelRichName" name="fudLabel">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
f&lt;span style=&quot; vertical-align:sub;&quot;&gt;ud&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabelUnitMeasure" name="fudUMLabel">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[Pa]
&lt;/body&gt;
&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEditValue" name="fudLEdit"/>
             </item>
             <item row="2" column="3">
              <widget class="QCheckBoxReadOnly" name="fudCheckBox">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabelRichName" name="fRareLabel">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
f&lt;span style=&quot; vertical-align:sub;&quot;&gt;rara&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabelUnitMeasure" name="fRareUMLabel">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[Pa]
&lt;/body&gt;
&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEditValue" name="fRareLEdit"/>
             </item>
             <item row="3" column="3">
              <widget class="QCheckBoxReadOnly" name="fRareCheckBox">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>225</width>
              <height>23</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelRichName</class>
   <extends>QLabel</extends>
   <header>qlabelrichname.h</header>
  </customwidget>
  <customwidget>
   <class>QLabelUnitMeasure</class>
   <extends>QLabel</extends>
   <header>qlabelunitmeasure.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditValue</class>
   <extends>QLineEdit</extends>
   <header>qlineeditvalue.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxReadOnly</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxreadonly.h</header>
  </customwidget>
  <customwidget>
   <class>QRadioButtonEnum</class>
   <extends>QRadioButton</extends>
   <header>qradiobuttonenum.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
